src/utils.c

```diff
@@ -256,6 +256,7 @@ is_scanner_only_pref (const char *pref)
       || !strcmp (pref, "log_plugins_name_at_load")
       || !strcmp (pref, "nasl_no_signature_check")
       || !strcmp (pref, "vendor_version") || !strcmp (pref, "drop_privileges")
+      || !strcmp (pref, "drop_privileges_user")
```

Comment:
Please add a man page entry for this new preference to https://github.com/greenbone/openvas/blob/master/doc/openvas.8.in

I'm also quite unhappy about the naming of this new preferences for the following reason:

There is also the existing `drop_privileges` preference (one line above) which is (if enabled) dropping the privileges of the whole `openvas` process. Due to the fact that `NULL` is passed to the `drop_privileges` here:

https://github.com/greenbone/openvas/blob/c84f5bd9780eeb96e2792c22ef6b7693d178344f/src/nasl_plugins.c#L243

the user `nobody` is always chosen:

https://github.com/greenbone/gvm-libs/blob/80b1e370b2e23d44ca8b43ff9bac0ef60b3eacf3/base/drop_privileges.c#L57

But due to the similarity of the naming for both preferences one might think that `drop_privileges_user` is the user which is used if `drop_privileges` is enabled which is not the case. In addition both preferences have a similar naming but are doing two completely different things:

- `drop_privileges` is dropping the privileges for the whole `openvas` process to `nobody`
- `drop_prvileges_user` is used in a single NASL function to drop the privileges to the specific user independent of the `drop_privileges` preferences

Currently i don't have a better suggestion for the naming but maybe some one of @greenbone/gvm-dev has some suggestions? If not let us discuss this next week.